backend/pom.xml

```diff
@@ -60,6 +61,29 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>mockwebserver</artifactId>
+            <version>4.9.1</version>
```

Comment:
Why do you not use the latest version 4.12.x?

Reply:
Thanks for the hint, will update.

Reply:
Added in commit 107ef608cdfb5d23c02f3cfd6ccd9fb05417bb35 ✅